scene/3d/baked_lightmap.cpp

```diff
@@ -1055,6 +1055,7 @@ float BakedLightmap::get_capture_cell_size() const {
 void BakedLightmap::set_extents(const Vector3 &p_extents) {
 	extents = p_extents;
 	update_gizmo();
+	_change_notify("extents");
 	_change_notify("bake_extents");
```

Comment:
Should `_change_notify("bake_extents");` below be removed? I guess the peroperty was renamed, making that line no longer relevant.

Reply:
That makes sense. I removed it from `set_extents()`